src/etsn20_lab4/str_search.java

```diff
@@ -6,9 +6,9 @@
 	public static void main(String[] args) {
 		
 	    Scanner userInput = new Scanner(System.in);
-	    System.out.println("Enter textfile to search (example.txt): ");
+	    System.out.println("Enter textfile to search (example.txt):... ");
 	    String textfile = userInput.nextLine();
-	    System.out.println("Enter pattern to search for: ");
+	    System.out.println("Enter pattern to search for:... ");
```

Comment:
Why Boolean instead of boolean?
Maybe this code belongs in two classes for extendability?
No comments at all.
The getResourceAsStream seems overly compilacated, maybe just use a File object?
The pattern and file input is flipped, should be <pattern> <file>
Good thing closing the resources, avoids memory leaks!